### DISCUSSION TITLE
Is it possible to remove the "slug" field from entry types?

### DISCUSSION THREAD

**Discussion by kruncher** [craftcms/cms#11642] [Ideas]
I often create entry types to hold data that will be accessed via GraphQL but are not pages on the resulting website.

The "slug" field serves no purpose but adds some complexity to what content editors see.

Is there a way to just turn the "slug" field off for an entry type?

**Comment by brandonkelly** [comment]
It’s not currently possible to hide it, so I’ve marked this as a feature request.
